### ISSUE TITLE
When I was running the training of crowdpose dataset, there were 'nan' and 'NaN or Inf found in input tensor' problems. I checked the data. This problem is caused by the disappearance or explosion of the gradient. After I adjusted the learning rate, this problem still can not be solved. Please help me.

### ISSUE THREAD

**Issue by LiPan123456789** [HRNet/DEKR#28]
Although it has this problem, it is still running normally. Is it right to continue running like this?
![image](https://user-images.githubusercontent.com/97578468/149125885-6062b21c-609a-47ed-8082-72f3d04cb3f0.png)

**Comment by Gengzigang** [comment]
Hi, I am sorry that I have not met this problem. You can reproduce our results in the paper using the cloned code without any change.
